locale/en/manageiq.po

```diff
@@ -61492,6 +61502,10 @@ msgstr ""
 msgid "Resize of VM has been successfully queued."
 msgstr ""
 
+#: ../app/javascript/components/vm-resize-form/vm-resize-form.jsx
+msgid "Resize of VM was cancelled by the user"
```

Comment:
This is an existing string that wasn't previously exposed for translation cc @MelsHyrule  🌮